.vscode/README.md

```diff
@@ -22,49 +22,100 @@ Install the following VSCode extensions:
 - Modern Fortran:
   https://marketplace.visualstudio.com/items?itemName=krvajalm.linter-gfortran
 
-Now, configure the VSCode files for the modflow6 directory.
-Open the `modflow6` directory in VSCode.
-Make sure that the setting "python.defaultInterpreterPath" points toward your Python executable.
+- vscode-modern-fortran-formatter:
+  https://marketplace.visualstudio.com/items?itemName=yukiuuh.vscode-modern-fortran-formatter
```

Comment:
Why do we need an additional extension?

Reply:
I was able to successfully integrate fprettify and VSCode with this extension.  I'm going to test with Modern Fortran and if I can achieve the desired result I'll remove vscode-modern-fortran-formatter.

Reply:
update:  I haven't been able to get Modern Fortran to work with the fprettify -c option in either of my test environments.  It is strange because the error displayed is an fprettify error but it is different depending on whether I provide a windows or unix style path.  It does work fine with options other than -c, including the full set in the config file.  Assuming I don't find a solution some options are:

- Modern Fortran as the only documented extension but with hardcoded options in the documentation that won't necessarily track with changes to to the config file.
- vscode-modern-fortran-formatter as the documented formatting extension with the -c option
- Describe both options in the document and leave it up to the user

Reply:
I would suggest to document only one of these options if both work fine.
@mjreno could you please open an issue at the [modern fortran extension repo](https://github.com/fortran-lang/vscode-fortran-support/issues) and describe which problems you are facing? Feel free to tag me.

Reply:
Will do @Hofer-Julian.